src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ShowPlan/Attributes.cs

```diff
@@ -283,5 +283,32 @@ public bool LongString
         private bool longString = false;
     }
 
+    public sealed class BetterValueAttribute: Attribute
```

Comment:
Don't usually comment on whitespace, but I think most would expect a space before the colon here.
`public sealed class BetterValueAttribute : Attribute`